packaging/hudi-presto-bundle/pom.xml

```diff
@@ -211,26 +209,8 @@
       <scope>compile</scope>
     </dependency>
 
-    <!--Guava needs to be shaded because HBase 1.2.3 depends on an earlier guava version i.e 12.0.1 and hits runtime
-    issues with the guava version present in Presto runtime-->
-    <dependency>
-      <groupId>com.google.guava</groupId>
-      <artifactId>guava</artifactId>
-      <version>12.0.1</version>
-      <scope>${presto.bundle.bootstrap.scope}</scope>
-    </dependency>
-
-    <!--commons-lang needs to be shaded because HBase 1.2.3 needs it at runtime, but Presto runtime does not have this
-    dependency-->
-    <dependency>
-      <groupId>commons-lang</groupId>
-      <artifactId>commons-lang</artifactId>
-      <version>2.6</version>
-      <scope>${presto.bundle.bootstrap.scope}</scope>
-    </dependency>
-
-    <!--protobuf needs to be shaded because HBase 1.2.3 needs it at runtime, but Presto runtime does not have this
-    dependency-->
+    <!--protobuf needs to be shaded because HBase 1.2.3 and our native reader needs it at runtime,
```

Comment:
We can also revisit protobuf dependency in a follow-up since HBase 2.4.9 also shaded protobuf IIRC.